### ISSUE TITLE
Improve the visibility of the verbatim identification

### ISSUE THREAD

**Issue by timrobertson100** [gbif/portal16#1202]
In many (hopefully most) cases the GBIF interpreted view of occurrences improves the identification by correcting typos, abbreviations and adding authorship. However, it causes real problems when the GBIF backbone does not hold a valid species or when some error has occurred and data are not organised correctly - it hides the visibility.

We should look to improve the visibility of this on the occurrence detail pages, specifically for cases where GBIF have changed the name beyond simply adding authorship.

**Comment by MortenHofft** [comment]
I'm not sure I agree, or at least I would like to understand more before changing anything.

I'm guessing this is based on that one user who stated that we do not show the information unless in diagnostics view, but this is not correct. We show it quite prominently in the header.

Before we start to "improve the visibility" of the verbatim name, then it might be worth understanding what went wrong in consuming the existing information.

![Screenshot 2019-08-12 at 10 06 48](https://user-images.githubusercontent.com/6759267/62869692-9c42d080-bd18-11e9-8474-28002ddd989a.png)
We show the original (verbatim) name as the header.
We show the interpreted name. And we show an orange warning stating that we couldn't find a match for this name.

**Comment by MortenHofft** [comment]
I've changed diagnostics to be the default. 

**History**
The previous website had a separate tab to show verbatim data.

When this page was redesigned we decided to mask verbatim data and give more emphasis to the interpreted data and instead allow for a comparison view for those interested in the audit trail.
This was based on the assumption, that most users cared about the cleaned/interpreted data and not the verbatim.

This perception has clearly changed in the last years. There is now a move towards making the audit trail more visible at all times. Following that I have now changed the occurrence detail page to show the comparison view as default. That means that the verbatim name is shown 
in the big fat orange header as well as in the table.
